apps/web/screens/OfficialJournalOfIceland/OJOISearch.tsx

```
@@ -414,13 +511,74 @@ const OJOISearch: CustomScreen<OJOISearchProps> = ({
+  let page: number | undefined
+  let pageSize: number | undefined
+
+  if (query.dagsFra && typeof query.dagsFra === 'string') {
+    const isValid = !isNaN(Date.parse(query.dagsFra))
+    if (isValid) {
+      dateFrom = new Date(query.dagsFra).toISOString().split('T')[0]
+    }
+  }
+
+  if (query.dagsTil && typeof query.dagsTil === 'string') {
+    const isValid = !isNaN(Date.parse(query.dagsTil))
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)